spec/features/users/navigation_spec.rb

```diff
@@ -25,9 +25,17 @@
     end
   end
   describe 'As a registered user' do
-    xit 'I see the same links as a visitor' do
-      user = User.create(email: "j@gmail.com", password:"password", role:0)
-      visit root_path
+    it 'I see the same links as a visitor' do
```

Comment:
I wonder if we can rephrase this slightly, since they see the same 1st 4 links, but then 2 links that are unique to a registered user. I think it may also be beneficial to add a test that we expect(page).to_not have_content for Login or Register?

Reply:
You are correct, again :)  I will change the wording to be more clear about what we are testing.